### PR TITLE
Newsweek clone

### PR DESCRIPTION
**Pull request by Hermela-Fikre**
In this project:

- I used bootstrap.
-I implemented various breakpoints depending on the element
- I used Html validation and stickler to enhance the code.

### CODE REVIEW

**Comment by mikenath223** [comment]
Pull request currently under review by Michgolden Ukeje (TSE)
Kindly exercise patience. 🐱

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Well done @Hermela-Fikre 👏👏

You've done quite well on this. 👍

However, there are a few minor changes I trust you will make before moving forward.
They are outlined below.

Good job once more. 👌

- [ ] _REQUIRED_ Make top-bar navigation links clickable. This includes the Sign-In/Subscribe element.
header-review

- [ ] OPTIONAL Add more margin to bottom footer.
footer

- [ ] _REQUIRED_ Remove comments in code. If you need to review a previous version of your code, use git history ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)).

File: style.css [hunk 1]
```diff
@@ -0,0 +1,644 @@
...
+  color: #fb9a94;
+}
+
+.fa-bars:hover {
+  color: #fb9a94;
+}
+
+/* .header__nav {
+  display: none;
+} */
```
Comment (mikenath223) [inline style.css]: Kindly remove these comments.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,713 @@
...
+        </div>
+        <nav class="header__nav row">
+            <img class="header1" src="img/header.png" alt="header">
+        </nav>
+    </header>
+    <main class="main container-fluid">
+        <div class="news-section row flex-wrap">
+            <div class="col-lg-7">
+                <div class="row">
+                    <div class="col-md-5 order-md-0"> <!-- Col 1 -->
```
Comment (mikenath223) [inline index.html]: Kindly remove the comment here also

File: index.html [hunk 3]
```diff
@@ -0,0 +1,713 @@
...
+                                <div class="position-relative mb-2">
+                                    <img class="w-100" src="https://d.newsweek.com/en/full/1571858/harry-meghan.jpg?w=397&h=265&l=0&t=0&q=90&f=b1ebfd0b58ce03f8ae98b9f1db9ef333" alt="Picture">
+                                    <span class="article__category py-2 px-3 font-weight-bold position-absolute r-txt text-uppercase">World</span>
+                                </div>
+                                <a class="link link--underline" href="#"><h3 class="news__article__header">What Meghan Markle and Prince Harry Did in Final Royal Appearances</h3></a>
+                                <p class="article__description mb-0">The Duke and Duchess of Sussex's farewell tour has included some show-stopping photo opportunities and predictable press negativity as they prepare to leave the royal family.</p>
+                            </article>
+                        </section>
+                    </div>     
+                    <div class="col-md-7 col-lg-7 d-flex flex-column"> <!-- Col 2-->
```
Comment (mikenath223) [inline index.html]: Kindly remove the comment here also.

**Review by MarioDena — APPROVED** [review]
Hello great work on making the requested changes! 👍 

## Your project is approved! 🏅 

## Optional changes

- [ ] There are some repeated rules on your CSS for example some `color` rules, this is generally a mandatory requirement but I'm listing it as optional since it was not mentioned on your previous review. This change is [OPTIONAL]

However I really recommend you make a review of your CSS since it will be mandatory on future projects. Here is an article if you need further help.  https://speckyboy.com/good-bad-css-practices/

Cheers!

## Code review by Mario Enrique Delgado Naranjo

**Pull request merged**